examples/python/gui_demo/app_context.py

```diff
@@ -160,3 +163,10 @@ def properties_of_component(self, component: daq.IComponent):
             return daq.IList()
 
         return component.all_properties if self.view_hidden_components else component.visible_properties
+
+    def on_core_event(self, sender: Optional[daq.IComponent], args: daq.IEventArgs):
+        if sender is not None and daq.IDevice.can_cast_from(sender) and args.event_name == "StatusChanged" and self.on_needs_refresh is not None:
+            core_event_args: daq.ICoreEventArgs = daq.ICoreEventArgs.cast_from(args)
+            has_connection_status = any(k == "ConnectionStatus" for k in core_event_args.parameters.keys())
```

Comment:
```suggestion
            has_connection_status = "ConnectionStatus" in core_event_args.parameters
```
Maybe you need parameters.keys(), I don't know if parameters is a normal python dict.

Reply:
On a second check, python will user "__iter__" operator if "__contains__" is not supported, so "in" operator should work